packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	BaseControl,
+	__experimentalUnitControl as UnitControl,
+} from '@wordpress/components';
+import { Platform } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+const isWeb = Platform.OS === 'web';
+
+const CSS_UNITS = [
+	{
+		value: 'px',
+		label: isWeb ? 'px' : __( 'Pixels (px)' ),
+		default: '2',
+	},
+	{
+		value: 'em',
+		label: isWeb ? 'em' : __( 'Relative to parent font size (em)' ),
+		default: '.2',
+	},
+	{
+		value: 'rem',
+		label: isWeb ? 'rem' : __( 'Relative to root font size (rem)' ),
+		default: '.2',
+	},
+];
+
+/**
+ * Control for letter-spacing.
+ *
+ * @param  {Object}   props                 Component props.
+ * @param  {string}   props.value           Currently selected letter-spacing.
+ * @param  {Function} props.onChange        Handles change in letter-spacing selection.
+ * @return {WPElement}                      Letter-spacing control.
+ */
+export default function LetterSpacingControl( { value, onChange } ) {
+	return (
+		<BaseControl className="block-editor-letter-spacing-control">
+			<BaseControl.VisualLabel>
+				{ __( 'Letter-spacing' ) }
+			</BaseControl.VisualLabel>
+			<UnitControl
+				aria-label={ __( 'Letter-spacing' ) }
+				value={ value }
+				__unstableInputWidth="60px"
+				units={ CSS_UNITS }
+				onChange={ onChange }
+			/>
+		</BaseControl>
```

Comment:
`UnitControl` supports a `label` prop; can we use that instead of `BaseControl` here? I think it'd make the code more semantic; and I think it's overall aligned with the component's intended usage. Furthermore, I think it'd allow us to drop `aria-label` (which I think uses the value of `label` as a fallback):

```suggestion
		<UnitControl
			aria-label={ __( 'Letter-spacing' ) }
			label={ __( 'Letter-spacing' ) }
			value={ value }
			__unstableInputWidth="60px"
			units={ CSS_UNITS }
			onChange={ onChange }
		/>
```

This will then also allow us to drop `packages/block-editor/src/components/letter-spacing-control/style.scss` 🙂

Reply:
The label for the font family control is also not at the same distance as the other controls in the typography panel, so the distances is already inconsistent. Which solution do we prefer?